docs/e2e/e2e-checks.md

```diff
@@ -26,29 +26,59 @@ Some highlights:
 - Parallelism limited on CI to ensure stable execution
 - Accessibility testing can be performed using the `@axe-core/playwright` package (https://playwright.dev/docs/accessibility-testing)
 
+### Running with Docker
+
+
+
+
+## How to Run Tests
+<table border="1" style="width:100%; text-align:center;">
+  <tr>
+    <th></th>
+    <th>Local Natively Without Docker</th>
+    <th>Local With Docker</th>
+    <th>CI / Github Actions</th>
+  </tr>
+  <tr>
+  <td>Location App is Running</td>
+  <td colspan="2" style="vertical-align:top;">Locally (*port 3000 in examples) </td>
+  <td>PR Preview Environment</td>
+  </tr>
+  <tr>
+    <td style="vertical-align:top;">With make commands</td>
+    <td style="vertical-align:top;">
+      From root folder:<br>
+      <ul style="list-style-position:inside; text-align:left;">
+        <li><code>make e2e-setup-native</code></li>
+        <li><code>make e2e-test APP_NAME=app BASE_URL=http://localhost:3000</code></li>
+        <li><code>make e2e-copy-report</code></li>
+      </ul>
+    </td>
+    <td style="vertical-align:top;">
+      From root folder:<br>
+      <ul style="list-style-position:inside; text-align:left;">
+        <li><code>make e2e-run APP_NAME=app BASE_URL=http://host.docker.internal:3000</code></li>
+        <br />
+        <em>* BASE_URL cannot use localhost</em>
+      </ul>
+    </td>
+    <td style="vertical-align:top;">
+      <em>* uses make commands <br /><br /> see the relevant <a href="../../.github/workflows/e2e-tests.yml">e2e Github Actions workflow file</a>
+    </em>
+    </td>
+  </tr>
+
+  <tr>
+    <td style="vertical-align:top;">Show Report</td>
+    <td colspan="2" style="vertical-align:top;">From the root: <br /><code>make e2e-show-report</code></td>
+    <td style="vertical-align:top;">View Artifacts of Github Actions job</td>
+  </tr>
+</table>
+
+- Running local with Docker is the preferred approach
+    - When running locally with Docker, the `playwright-report` will be copied to your local `./e2e/` folder
+- For all local runs, your application needs to be running
```

Comment:
This table is still hard to read in the code, the intent of markdown is that it should be human readable. I don't think a table is needed here. Just write this in prose. Something like

```suggestion
## Run end-to-end tests

First, make sure the application you want to test is running.

Then, run end-to-end tests using Docker with

``
make e2e-test
``

## Run end-to-end tests natively

First, make sure the application you want to test is running.

To run end-to-end tests natively, first install Playwright with

``bash
make e2e-setup-native
``

Then run the tests with

``bash
make e2e-test-native
``
```